packages/@navikt/popover/src/popover.stories.tsx

```diff
@@ -3,11 +3,10 @@ import Popover from './popover';
 
 export default { title: '@navikt/popover' };
 
-export const normal = () => {
-    const [isOpen, setIsOpen] = React.useState(false);
```

Comment:
Var det meningen å fjerne dette?

Reply:
ja, jeg brukte hooks på feil måte. Fant en fremgangsmåte å bruke hooks i stories på, men ble noe rart med koden i storybook-viewet. Ved å lage to stories får vi eksempel på popover i ulike states, samt snapshot og screenshot av begge